torch/fx/proxy.py

```diff
@@ -128,6 +183,7 @@ def __getattr__(self, k) -> 'Attribute':
         return Attribute(self, k)
 
     def __call__(self, *args, **kwargs) -> 'Proxy':
+        _create_friendly_names(args, kwargs, 1)
```

Comment:
seems like every call to create_proxy needs to have a `_create_friendly_names` precede. That indicates to me `create_proxy` should be the one handling this.